src/lib/components/consignments/components/__tests__/artwork-consignment-todo-tests.tsx

```diff
@@ -0,0 +1,20 @@
+import "react-native"
+
+import * as React from "react"
+import * as renderer from "react-test-renderer"
+
+import { allStates } from "../../__stories__/consignments-todo.story"
+
+import TODO from "../artwork-consignment-todo"
+
+describe("For different states", () => {
+  for (const key in allStates) {
+    if (allStates.hasOwnProperty(key)) {
+      const state = allStates[key]
+      it(`Looks right ${key}`, () => {
+        const todo = renderer.create(<TODO {...state}/>)
+        expect(todo.toJSON()).toMatchSnapshot()
+      })
+    }
+  }
+})
```

Comment:
I started #559 as a part of understanding that we don't have good tools for writing behaviour tests